src/aerie_cli/aerie_host.py

```diff
@@ -278,6 +290,28 @@ def authenticate(self, username: str, password: str = None):
         if not self.check_auth():
             raise RuntimeError(f"Failed to open session")
 
+    def check_aerie_version(self) -> None:
+        """Assert that the Aerie host is a compatible version
+
+        Raises a `RuntimeError` if the host appears to be incompatible.
+        """
+
+        resp = self.session.get(self.gateway_url + "/version")
+
+        try:
+            resp_json = process_gateway_response(resp)
+            host_version = resp_json["version"]
+        except (RuntimeError, KeyError):
+            # If the Gateway responded, the route doesn't exist
+            if resp.text and "Aerie Gateway" in resp.text:
+                raise AerieHostVersionError("Incompatible Aerie version: host version unknown")
```

Comment:
Could you share an example response for when you try to hit this endpoint on an Aerie version without a `version` endpoint?

Reply:
![image](https://github.com/user-attachments/assets/1ae717eb-af18-4161-8478-08a5f4907576)

A `get` request to an unknown route on the Gateway just gets this regular UI page back (interesting that it doesn't get a 404 or even a redirect, but 🤷)

Reply:
Huh. Alright I see why you're checking for `Aerie Gateway` in the response text now